traffic/config.ini

```diff
@@ -4,25 +4,30 @@ broker_address = test.mosquitto.org
 port= 1883
 username= username
```

Comment:
see my comment below, let's comment this if you are not using it now, but let's support the code to check if defined or not to use them to connect

Reply:
done the username and password are empty in the config file